projects/schematics/src/migrations/4_0/constructor-deprecations/data/configurator-form.component.migration.ts

```diff
@@ -0,0 +1,19 @@
+import {
+  CONFIGURATOR_FORM_COMPONENT,
+  CONFIGURATOR_STOREFRONT_UTILS_SERVICE,
+  SPARTACUS_PRODUCT_CONFIGURATOR_RULEBASED,
+} from '../../../../shared/constants';
+import { ConstructorDeprecation } from '../../../../shared/utils/file-utils';
+
+export const CONFIGURATOR_FORM_COMPONENT_MIGRATION: ConstructorDeprecation = {
+  //feature-libs/product-configurator/rulebased/components/form/configurator-form.component.ts
+  class: CONFIGURATOR_FORM_COMPONENT,
+  importPath: SPARTACUS_PRODUCT_CONFIGURATOR_RULEBASED,
+  deprecatedParams: [],
```

Comment:
This has to be specified, otherwise the update mechanism won't pick the class. Did you unit-test this migration?

Reply:
Here nothing was deprecated, new dependency was added

Reply:
Done!